base/process.jl

```diff
@@ -311,7 +311,9 @@ macro setup_stdio()
         in,out,err = stdios
         if isa(stdios[1], Pipe)
             if stdios[1].handle == C_NULL
```

Comment:
reminder (this should be using the test `isopen(stdios[1])`, now that we have that field)